config/locales/en.yml

```
@@ -242,7 +242,7 @@ en:
     confirm_email_success: Email address confirmed
     contact_details_form: Provide contact details for support
     date_settings: Are you asking for someone's date of birth?
-    declaration_form: Add a declaration
+    declaration_form: Add a declaration for people to agree to 
```

There's a space at the end of this line which is making one of the tests complain:

```suggestion
    declaration_form: Add a declaration for people to agree to
```